packages/imba/src/imba/setupVite.imba

```diff
@@ -0,0 +1,47 @@
+# imba$stdlib=1
+import np from 'path'
+import * as Vite from "vite"
```

Comment:
Won't this end up importing vite whenever you run imba in node? Or do you know if it is removed by tree-shaking in esbuild?

Reply:
Sorry for the late reply 🙂 There seem to be issues related to this 

Running a script complains about
```
❯ imba index.imba
▲ [WARNING] "esbuild" should be marked as external for use with "require.resolve" [require-resolve-not-external]

    ../imba/packages/imba/node_modules/esbuild/lib/main.js:1718:36:
      1718 │   const libMainJS = require.resolve("esbuild");
           ╵                                     ~~~~~~~~~

✘ [ERROR] No loader is configured for ".node" files: ../imba/packages/imba/node_modules/fsevents/fsevents.node

    ../imba/packages/imba/node_modules/fsevents/fsevents.js:13:23:
      13 │ const Native = require("./fsevents.node");
         ╵                        ~~~~~~~~~~~~~~~~~

× failed with 1 errors
```

Reply:
I removed `setupVite` from imba directly into `imba/plugin`. It was indeed causing the issue :) Nice catch